_rules/important-letter-spacing-wide-enough-24afc2.md

```diff
@@ -16,74 +16,74 @@ input_aspects:
 acknowledgments:
   authors:
     - Jean-Yves Moyen
+  previous_authors:
     - Jey Nandakumar
   funding:
     - WAI-Tools
 ---
 
 ## Applicability
 
-This rule applies to any [HTML element][] that is [visible][] and for which the `style` attribute [declares][declared] the [letter-spacing][] CSS property.
+This rule applies to any [visible][] [text node][] child of an [HTML element][], when all the following are true for the `letter-spacing` property of the element:
+
+- the [specified][] value is [declared][] in a `style` attribute; and
+- the [computed][] value is [important][].
 
 ## Expectation
 
-For each test target, at least one of the following is true:
+For each test target, the [computed][] value of its `letter-spacing` property is at least 0.12 times the [computed][] value of its `font-size` property.
```

Comment:
So... bad news. I think I found a new problem with this. Or maybe, I found that the old problem hasn't been solved. We were previously assuming that you could get the font-size from the element with the style attribute, we're now assuming you can get font-size from the parent of the text node. I don't think that's true though. Someone could use pseudo-elements like `::first-line` to adjust font-size, and this rule would not spot that.

Reply:
There is an assumption precisely to handle these cases.
In the current version, the assumption is here for the `line-height` rule, now copied in all three.